### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by HieuCR7**
hh

### CODE REVIEW

**Comment by HieuCR7** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (hhhhuy112) [inline app/helpers/application_helper.rb]: I18n for  "Ruby on Rails Tutorial Sample App"

**Comment by hhhhuy112** [comment]
commented

**Comment by HieuCR7** [comment]
ready

**Comment by hhhhuy112** [comment]
commented
Please rebase 4 commit

**Comment by HieuCR7** [comment]
chapter4

**Comment by HieuCR7** [comment]
ready

**Comment by HieuCR7** [comment]
chapter4

**Comment by HieuCR7** [comment]
ready

**Comment by VuThiTranVan** [comment]
Commented
Sao 5 commit lun vậy em, rebase -i thành 1 commit thôi nhé

**Pull request closed without merging**